### PR TITLE
Artillery vehicle to destroy artillery mission

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [ ] Enhancement
4. [x] Miscellaneous

### What have you changed and why?
Information:

Added more vehicles to the artillery pool in the destroy artillery mission

Added extra fallbacks in some addition templates to handle missing values

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/functions/Missions/fn_DES_Artillery.sqf [hunk 1]
```diff
@@ -10,10 +10,17 @@ Info("Artillery mission init.");
...
+private _difficult = if (random 10 < tierWar) then {true} else {false};
 private _sideX = if (sidesX getVariable [_markerX, sideUnknown] == Occupants) then {Occupants} else {Invaders};
 private _faction = Faction(_sideX);
 
+diag_log _faction;
+diag_log _faction;
+diag_log _faction;
+diag_log _sideX;
+diag_log _sideX;
+diag_log _sideX;
```
Comment (Silence-112) [inline A3A/addons/core/functions/Missions/fn_DES_Artillery.sqf]: Try not to leave debug logging unless it is set up properly and has a purpose

**Review by Silence-112 — COMMENTED** [review]
What purpose do these changes serve?

**Comment by wersal454** [comment]
> What purpose do these changes serve?

Thought it would be right to add more vehicle variety to the mission

**Comment by Silence-112** [comment]
Did some improvements in my last commit, feel free to test any edge cases I might have missed

**Review by Silence-112 — APPROVED** [review]
Any potential edge cases should be handled by the fallback/redundancy code

lgtm
